tests/CodeBehind/BuildTests/MainActivityNotPartial.cs

```diff
@@ -14,7 +14,8 @@ protected override void OnCreate (Bundle savedInstanceState)
 			base.OnCreate (savedInstanceState);
 
 			// Set our view from the "main" layout resource
-			Binding.Main items = SetContentView <Binding.Main> ();
```

Comment:
You'd previously mentioned being sad about losing the `SetContentView<T>()` method.

We could perhaps instead do it as an extension method. You'd still need an extra `this.` to invoke it, but it could still exist...

```csharp
var items = this.SetContentView<Binding.Main>();
```

Reply:
Nah, let's just bury the idea instead of making it uglier :)